addons/portal_rating/models/rating_rating.py

```diff
@@ -54,3 +55,15 @@ def _synchronize_publisher_values(self, values):
             if not values.get('publisher_id'):
                 values['publisher_id'] = self.env.user.partner_id.id
         return values
+
+    def _to_store_defaults(self):
+        return super()._to_store_defaults() + (
+            [
+                Store.One("message_id", []),
+                Store.One("publisher_id", ["name", "is_company", "user", "write_date"]),
+                Store.Attr("publisher_comment"),
+                Store.Attr("publisher_datetime"),
```

Comment:
Actual field name don't need the Attr wrapping if there is no other param necessary.